packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -111,7 +98,7 @@ export const getStyles = memoizeFunction((
       color: checkmarkFontColorCheckedDisabled,
     },
     text: {
-      margin: '0 4px'
+      marginRight: '8px'

```

Comment:
The code looks good. I have one question about how this would look in case of RTL. I assumed that was the reason the margins were symmetric before.